utils/build/virtual_machine/provisions/auto-inject-ld-preload/main.c

```diff
@@ -0,0 +1,7 @@
+#include <stdio.h>
+
+int puts(const char *__s)
+{
+    return printf("New puts\n");
+    
+}
```

Comment:
I'm not sure I understand why this is needed. Do you mind adding a comment?

Reply:
It only for ld preload test. We are adding thing to ld preload before install dd software. I'll add a comment